src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,33 @@
+import { Link } from 'react-router-dom';
+
+import { useWindowSize } from '../../app/hooks';
+import { Logo } from '../Logo';
+import { ToTopButton } from '../ToTopButton';
+import './Footer.scss';
+
+export const Footer: React.FC = () => {
+  const { width } = useWindowSize();
+
+  return (
+    <footer className="footer">
+      <div className="footer__container">
+        {width > 640 ? (
```

Comment:
It would be simplier to have such ternary operator:
`<Logo type={width > 640 ? "big" : "small"} />`